awx/main/tests/functional/test_inventory_source_injectors.py

```diff
@@ -258,6 +258,8 @@ def test_inventory_update_injected_content(this_kind, script_or_plugin, inventor
     use_plugin = bool(script_or_plugin == 'plugins')
     if use_plugin and InventorySource.injectors[this_kind].plugin_name is None:
         pytest.skip('Use of inventory plugin is not enabled for this source')
+    elif not use_plugin and InventorySource.injectors[this_kind].plugin_name is not None:
```

Comment:
> `InventorySource.injectors[this_kind].plugin_name` 

seems like this would need to be `.get('plugin_name', None)` - don't think the plugin_name is always defined (see [vmware](https://github.com/ansible/awx/pull/6088/files#diff-131ca73c9ddeed5719e3ac78360883ccR2200), for example)

Reply:
but the common base class `PluginFileInjector` sets `plugin_name = None`, so testing if it's None should work right. I think.

Reply:
wait, now you have me questioning if we are running all the script cases we should be.